packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.ts

```diff
@@ -96,10 +97,15 @@ async function filterDonorsInGroup(
 
 function filterAppointmentsForDonors(
   appointments: FunctionsApi.AppointmentApiEntry[],
-  donors: DbDonor[]
+  donors: DbDonor[],
+  assigningCoordinator: string
 ) {
   const donorIds = new Set(donors.map((donor) => donor.id));
   return appointments.filter(
-    (appointment) => appointment.donorId && donorIds.has(appointment.donorId)
+    (appointment) =>
+      appointment.donorId &&
+      (donorIds.has(appointment.donorId) ||
+        (appointment.donorId === "manual" &&
+          appointment.assigningCoordinator === assigningCoordinator))
```

Comment:
Adding && and || in the same condition is bad practice.

Instead of using a one-liner, split this to if and else if...